### PR TITLE
docs: adding height width attrs to images

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

Note: this is the second half of the width/height attrs PR, first one fixed these in the `/elements/**/docs` folder

1. added `height` and `width` attrs to images 
2. fixed `data-helmet` on js/style tags per page in the `docs/` directory
3. cleaned up some embedded svg use 
4. fixed margin around toc at mobile sized viewports

## Testing Instructions

1.

### CODE REVIEW

File: docs/accessibility/accessibility-tools.md [hunk 1]
```diff
@@ -58,7 +58,10 @@ Google's Chrome and Microsoft's Edge browsers have several accessibility tools b
 
 <figure>
   <uxdot-example width-adjustment="900px">
-    <img src="/assets/accessibility/lighthouse-audit.png" alt="Screenshot of a Lighthouse autdit with a score of 100%.">
+    <img src="/assets/accessibility/lighthouse-audit.png"
+      alt="Screenshot of a Lighthouse autdit with a score of 100%."
+      width="900"
+      height="251">
```
Comment (adamjohnson) [inline docs/accessibility/accessibility-tools.md:64]: Make sure to have these attributes properly aligned ala [wiki guidelines](https://github.com/RedHat-UX/red-hat-design-system/wiki/HTML-Formatting#attributes).

That goes for this `img` tag and the rest of the images in this PR.

Comment (adamjohnson) [reply docs/accessibility/accessibility-tools.md:64]: Side note: does this alignment need to be added to the 1st PR in this series?

**Review by adamjohnson — APPROVED** [review]
Mostly alignment of attributes. Left-alignment Gets Tags Modified

File: docs/about/index.md [hunk 2]
```diff
@@ -85,7 +85,10 @@ and adapts to meet new challenges.
 <rh-cta href="https://www.redhat.com/en/about/brand/standards">Learn about the Red Hat brand</rh-cta>
 
 <uxdot-example variant="full" no-border alignment="left" width-adjustment="1140px">
- <img src="../assets/about/about-rhds-brand-standards.png" alt="the words 'brand standards' framed by elements and shapes in ReEd Hat colors">
+ <img src="../assets/about/about-rhds-brand-standards.png"
+      alt="the words 'brand standards' framed by elements and shapes in ReEd Hat colors"
```
[the remark this answers is not in the document]
Comment (zeroedin) [reply docs/about/index.md:89]: Good catch, looks like that is a issue outside these changes 💸

File: docs/accessibility/accessibility-tools.md (hunk 1 above)
Comment (zeroedin) [reply docs/accessibility/accessibility-tools.md:64]: From the guidelines:
> Attributes should be indented by at least two units.

What are you seeing different?

Comment (adamjohnson) [reply docs/accessibility/accessibility-tools.md:64]: > Columnar Attributes
Attributes _may_ line up with the first inline attribute, so long as they are indented by as least two units. This is a matter of taste which can produce more beautiful and easier to understand templates.

In a recent PR, I aligned image attrs to look like they do here and Benny aligned them in a vertically straight column in a subsequent commit.

**Pull request merged**